src/Accounts/Accounts/CommonModule/ContextAdapter.cs

```diff
@@ -269,7 +279,12 @@ internal async Task<IAccessToken> AuthorizeRequest(IAzureContext context, HttpRe
                     var tokenAudience = tokenAudienceConverter.Invoke(info.CurEnvEndpointResourceId, info.CurEnvEndpointSuffix, info.BaseEnvEndpointResourceId, info.BaseEnvEndpointSuffix, request.RequestUri);
                     endpointResourceIdKey = tokenAudience ?? endpointResourceIdKey;
                 }
-                var authToken = _authenticator.Authenticate(context.Account, context.Environment, context.Tenant.Id, null, "Never", null, endpointResourceIdKey);
+                var optionalParameters = new Dictionary<string, object>() { { AuthenticationFactory.ResourceIdParameterName, endpointResourceIdKey } };
+                if (extensibleParameters != null && extensibleParameters.ContainsKey(AuthenticationFactory.CmdletContextParameterName))
+                {
+                    optionalParameters.Add(AuthenticationFactory.CmdletContextParameterName, extensibleParameters[AuthenticationFactory.CmdletContextParameterName]);
+                }
+                var authToken = _authenticator.Authenticate(context.Account, context.Environment, context.Tenant.Id, null, "Never", null, optionalParameters);
```

Comment:
Could you please explain the change here?

Reply:
Remove generated module related change as more changes are required to enable the function in generated modules